pkg/identity/ciprovider/principal.go

```diff
@@ -81,7 +82,7 @@ func applyTemplateOrReplace(extValueTemplate string, tokenClaims map[string]stri
 	}
 	claimValue, ok := mergedData[extValueTemplate]
 	if !ok {
-		return "", fmt.Errorf("value <%s> not present in either claims or defaults", extValueTemplate)
+		return "", fmt.Errorf("value <%s> not present in either claims or defaults. Issuer: %s", extValueTemplate, issuer)
```

Comment:
Any more logging we can do? What about which extension we're trying to write to?

Reply:
I was worried about logging any sensitive information, but I can add the extension for sure. wdyt?

Reply:
Yea, the name of the ext seems fine. It'll help us reproduce this once we know which issuer and which claim is causing the issue.